package.json

```diff
@@ -22,7 +22,7 @@
     "react": "^15.4.1",
     "react-dom": "^15.4.1",
     "react-tabs": "^0.8.2",
-    "tachyons-egghead": "^1.1.5",
+    "tachyons-egghead": "eggheadio/tachyons-egghead#textfield",
```

Comment:
@trevordmiller I opened a PR to officially bump `tachyons-egghead` to 1.1.6... I aimed `package.json` directly to the branch so I could test both at the same time and then forgot to put it back at first (though I thought I re-pushed here)